packages/gatsby/src/utils/api-node-docs.js

```diff
@@ -78,7 +78,12 @@ exports.createPages = true
 exports.createPagesStatefully = true
 
 /**
- * Extension point to tell plugins to source nodes.
+ * Extension point to tell plugins to source nodes. This API is called during
+ * the Gatsby bootstrap sequence. Source plugins use this hook to create nodes.
+ * This API is called exactly once per plugin (and once for your site's
+ * `gatsby-config.js` file). If you define this hook in `gatsby-config.js` it
```

Comment:
Do you mean in `gatsby-node.js`?

Reply:
Oooh, nice catch. Yep, that’ll be a mistake, will fix and push an update.